docs/source/reference/yaml-spec.rst

```diff
@@ -92,10 +92,16 @@ Available fields:
       # If unspecified, defaults to False (on-demand instances).
       use_spot: False
 
-      # The recovery strategy for spot jobs (optional).
-      # `use_spot` must be True for this to have any effect. For now, only
-      # `FAILOVER` strategy is supported.
-      spot_recovery: none
+      # The recovery strategy for managed jobs (optional).
```

Comment:
For discussion: How about we hide this field since the default EAGER_NEXT_REGION has been working well?